packages/ckeditor5-table/src/tablecolumnresize/tablecolumnresizeediting.js

```diff
@@ -401,35 +390,35 @@ export default class TableColumnResizeEditing extends Plugin {
 
 		// At this point we change only the editor view - we don't want other users to see our changes yet,
 		// so we can't apply them in the model.
-		editingView.change( writer => this._applyResizingAttributesToTable( writer, target, viewTable ) );
+		editingView.change( writer => this._applyResizingAttributesToTable( writer, viewTable ) );
 	}
 
 	/**
-	 * Calculate the dom column widths. It is done by taking the width of the widest cell
-	 * from each table column (we rely on the TableWalker to determine
-	 * which column the cell belongs to).
+	 * Calculates the dom column widths. It is done by taking the width of the widest cell
+	 * from each table column (we rely on the  {@link module:table/tablewalker~TableWalker}
+	 * to determine which column the cell belongs to).
 	 *
 	 * @private
 	 * @param {module:engine/model/element~Element} modelTable A table which columns should be measured.
-	 * @returns {Array.<Number>} Widths expressed in pixels (without unit).
+	 * @returns {Array.<Number>} Column widths expressed in pixels (without unit).
 	 */
 	_calculateDomColumnWidths( modelTable ) {
 		const editor = this.editor;
-		const columnWidthsInPx = Array( getNumberOfColumn( modelTable, editor ) );
+		const columnWidthsInPx = Array( editor.plugins.get( 'TableUtils' ).getColumns( modelTable ) );
```

Comment:
`editor.plugins.get( 'TableUtils' ).getColumns` is used 3 times in this file, maybe this could be simplified by importing this function at the beginning of the file? :thinking:

Reply:
We can't do an explicit `import { getColumns } from '../tableutils.js';` because this function is not directly exported, only as a part of the class. And since the references are scattered among various methods, we'd have to add this function to the topmost level of the `TableColumnResizeEditing` class, which seems a bit messy. Perhaps I just don't see something, so if you or @mlewand have an idea how to do it I'm open for suggestions :open\_hands:

Reply:
Given that we're using `TableUtils` extensively in this plugin I think we could assign a reference to the instance of this plugin. It make sense to make it a private member.

We could obtain it and assign at a time of construction.

What's more interesting is that even though we use it that intensively, we require it only [implicitly though `TableEditing` plugin](https://github.com/ckeditor/ckeditor5/blob/806508a60599d234a0f38782969992fbba82143b/packages/ckeditor5-table/src/tablecolumnresize/tablecolumnresizeediting.js#L53). I think it make sense to make to require it explicitly in case something changes in TableEditing plugin implementation.